static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -68,18 +69,34 @@ export function TraceIssueEvent({event}: TraceIssueEventProps) {
   );
 }
 
+// This function tries to imitate what getTitle() from utils.events does.
+// In that module, the data comes from the issues endpoint while in here
+// we grab the data from the events endpoint. A larger effort is
+// required in order to use that function directly since the data between
+// the two endpoint is slightly different.
+// For instance, the events endpoint could include a _metadata dict with
+// the title, subtitle and message.
+// We could also make another call to the issues endpoint  to fetch the metadata,
+// however, we currently don't support it and it is extremely slow
 function getTitleSubtitleMessage(event: TimelineEvent) {
   let title;
-  // XXX: This is not fully correct but it will make this first PR easier to review
+  // XXX: This is not fully correct but it will make following PRs easier to review
   const subtitle = event.transaction;
   let message = event.message;
-  if (event['event.type'] === 'error') {
-    title = event.title.split(':')[0];
-  } else {
-    title = event.title;
-    // It is suspected that this value is calculated somewhere in Relay
-    // and we deconstruct it here to match what the Issue details page shows
-    message = event.message.replace(event.transaction, '').replace(title, '');
+  try {
+    title = event.title.trimEnd();
+    if (event['event.type'] === 'error') {
+      if (title[title.length - 1] !== ':') {
```

Comment:
These 3 lines are to handle this use case:
<img width="583" alt="image" src="https://github.com/getsentry/sentry/assets/44410/d7623c55-e09e-4606-be68-452ac5685487">